### PR TITLE
Package kkmarkdown.1.0.0

### PR DESCRIPTION
**Pull request by kkeundotnet**
kkmarkdown is (yet another) markdown engine that is,

* to avoid XSS attack, by supporting limited markdown syntax,
* to run the same markdown engine in both server and client sides, thanks to js_of_ocaml.

### CODE REVIEW

File: packages/kkmarkdown/kkmarkdown.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* To run the same markdown engine in both server and client sides, thanks to js_of_ocaml.
+"""
+maintainer: "kkeundotnet <kkeundotnet@gmail.com>"
+authors: "kkeundotnet <kkeundotnet@gmail.com>"
+license: "Unlicense"
+homepage: "https://github.com/kkeundotnet/kkmarkdown"
+bug-reports: "https://github.com/kkeundotnet/kkmarkdown/issues"
+dev-repo: "git+https://github.com/kkeundotnet/kkmarkdown.git"
+depends: [
+  "dune" {build}
```
Comment (mseri) [inline packages/kkmarkdown/kkmarkdown.1.0.0/opam:18]:
```suggestion
  "dune" {>= "3.6"}
```
According to the dune-project file. Note that due to a bug in dune (https://github.com/ocaml/dune/issues/2147), it should never be tagged as a build dependency.

File: packages/kkmarkdown/kkmarkdown.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+name: "kkmarkdown"
```
Comment (mseri) [inline packages/kkmarkdown/kkmarkdown.1.0.0/opam:2]:
```suggestion
```

File: packages/kkmarkdown/kkmarkdown.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+version: "1.0.0"
```
Comment (mseri) [inline packages/kkmarkdown/kkmarkdown.1.0.0/opam:3]:
```suggestion
```

File: packages/kkmarkdown/kkmarkdown.1.0.0/opam (hunk 1 above)
Comment (kkeundotnet) [reply packages/kkmarkdown/kkmarkdown.1.0.0/opam]: Thank you. Good to know that.

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**